File: 백준/Bronze/30008. 준영이의 등급/준영이의 등급.cs
string[] str = Console.ReadLine().Split();

int n = int.Parse(str[0]);  //학생 수
int k = int.Parse(str[1]);  //과목 수

int[] ranks = Array.ConvertAll(Console.ReadLine().Split(), int.Parse);

for (int i = 0; i < k; i++)
{
    int P = (ranks[i] * 100) / n;

    if (P <= 4) Console.Write(1 + " ");
    else if (P <= 11) Console.Write(2 + " ");
    else if (P <= 23) Console.Write(3 + " ");
    else if (P <= 40) Console.Write(4 + " ");
    else if (P <= 60) Console.Write(5 + " ");
    else if (P <= 77) Console.Write(6 + " ");
    else if (P <= 89) Console.Write(7 + " ");
    else if (P <= 96) Console.Write(8 + " ");
    else Console.Write(9 + " ");
}

File: 백준/Bronze/30008. 준영이의 등급/준영이의 등급.py
N, K = map(int, input().split())
G = list(map(int, input().split()))

for i in G:
    P = (i * 100) // N
    if 0 <= P <= 4:
        print(1, end=' ')
    elif P <= 11:
        print(2, end=' ')
    elif P <= 23:
        print(3, end=' ')
    elif P <= 40:
        print(4, end=' ')
    elif P <= 60:
        print(5, end=' ')
    elif P <= 77:
        print(6, end=' ')
    elif P <= 89:
        print(7, end=' ')
    elif P <= 96:
       print(8, end=' ')
    elif P <= 100:
        print(9, end=' ')